### PR TITLE
[Admin] General UI improvements

### PR DESCRIPTION
**Pull request by pjedrzejewski**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT

Please do not review yet. ;)

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/config/grids/promotion.yml [hunk 1]
```diff
@@ -10,15 +10,33 @@ sylius_grid:
...
-                    type: string
-                    label: sylius.ui.used
+                    path: .
+                    options:
+                        template: SyliusAdminBundle:Grid/Field:nameAndDescription.html.twig
+                couponBased:
+                    type: twig
+                    label: sylius.ui.coupons
+                    options:
+                        template: SyliusAdminBundle:Grid/Field:yesNo.html.twig
```
Comment (songecko) [inline src/Sylius/Bundle/AdminBundle/Resources/config/grids/promotion.yml]: @pjedrzejewski What are your opinion about to use the name of the template like the field type. For example, yesNo.html.twig => boolean.html.twig, etc...

File: src/Sylius/Behat/Page/Admin/Country/UpdatePage.php [hunk 2]
```diff
@@ -82,10 +82,14 @@ public function removeProvince($provinceName)
             $provinces = $this->getElement('provinces');
 
             $item = $provinces
-                ->find('css', 'div[data-form-collection="item"] input[value="'.$provinceName.'"]')
+                ->find('css', sprintf('div[data-form-collection="item"] input[value="%s"]', $provinceName))
+                ->getParent()
+                ->getParent()
+                ->getParent()
```
Comment (pjedrzejewski) [inline src/Sylius/Behat/Page/Admin/Country/UpdatePage.php]: XD

Comment (pamil) [inline src/Sylius/Behat/Page/Admin/Country/UpdatePage.php]: Top kek

File: src/Sylius/Behat/Page/Admin/Country/UpdatePage.php [hunk 3]
```diff
@@ -82,10 +82,14 @@ public function removeProvince($provinceName)
...
                 ->getParent()
                 ->getParent()
```
Comment (pamil) [inline src/Sylius/Behat/Page/Admin/Country/UpdatePage.php]: It's even longer :bug: :bug:

File: src/Sylius/Behat/Page/Admin/Customer/CreatePage.php [hunk 4]
```diff
@@ -66,9 +66,12 @@ public function chooseGender($gender)
         $this->getDocument()->selectFieldOption('Gender', $gender);
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function selectCreateAccount()
     {
-        $this->getDocument()->checkField('Create account?');
+        $this->getDocument()->find('css', 'label[for=sylius_customer_create_user]')->click();
```
Comment (Arminek) [inline src/Sylius/Behat/Page/Admin/Customer/CreatePage.php]: 🎉

File: src/Sylius/Bundle/AdminBundle/Resources/config/routing/product_attribute.yml [hunk 5]
```diff
@@ -30,6 +30,7 @@ sylius_admin_product_attribute_create:
             template: SyliusAdminBundle:Crud:create.html.twig
             redirect: sylius_admin_product_attribute_update
             vars:
+                subheader: sylius.ui.manage_attributes_of_your_produts
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/AdminBundle/Resources/config/routing/product_attribute.yml]: produ**c**ts

**Comment by michalmarcinkowski** [comment]
Looks nice! :art: :sparkles:

**Comment by michalmarcinkowski** [comment]
Good job Paweł! 🎆

**Pull request merged**